### ISSUE TITLE
Issue wit target version upgrading to  30  Caused by: com.agog.mathdisplay.parse.MathDisplayException: Error initializing FreeType. 

### ISSUE THREAD

**Issue by iGuru-iOS** [gregcockroft/AndroidMath#14]
Caused by: android.view.InflateException: Binary XML file line #28 in .XMl : Error inflating class com.agog.mathdisplay.MTMathView
     Caused by: java.lang.reflect.InvocationTargetException

**Comment by Aiushtha** [comment]
haha,Hello friends, I also encountered this problem. In order to rush to go online, I finally switched the clatexmath library module after the freetype initialization failed.

> Caused by: android.view.InflateException: Binary XML file line #28 in .XMl : Error inflating class com.agog.mathdisplay.MTMathView Caused by: java.lang.reflect.InvocationTargetException


haha ,Hello friends, I also encountered this problem. In order to rush to go online, I finally switched the ' cLaTeXmath ' library module after the freetype initialization failed.
